packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -313,6 +313,26 @@ export interface VideoGalleryProps {
    * This callback is to mute a remote participant
    */
   onMuteParticipant?: (userId: string) => Promise<void>;
+  /* @conditional-compile-remove(media-access) */
+  /**
+   * This callback is to forbid audio for a remote participant
+   */
+  onForbidAudio?: (userIds: string[]) => Promise<void>;
```

Comment:
Same as comment above we might want this to be `onToggleX` as this expands and already large API by a lot